src/blocks/imxrt1021/iomuxc.rs

```diff
@@ -16752,10 +16752,10 @@ pub mod LPSPI1_SDO_SELECT_INPUT {
         pub mod R {}
         pub mod W {}
         pub mod RW {
-            #[doc = "Selecting Pad: GPIO_AD_04 for Mode: ALT0"]
-            pub const GPIO_AD_04_ALT0: u32 = 0;
-            #[doc = "Selecting Pad: GPIO_SD_06 for Mode: ALT2"]
-            pub const GPIO_SD_06_ALT2: u32 = 0x01;
+            #[doc = "Selecting Pad: GPIO_SD_B0_04 for Mode: ALT4"]
+            pub const GPIO_SD_B0_04_ALT4: u32 = 0;
```

Comment:
This seems like a suspicious change, but is exactly the problem you noted existing without this PR correct?

The ref manual suggests the original values were correct and the updated values here are wrong

Reply:
I have more spot checks like this to do. We're commenting on the **1021** IOMUXC module, and this one matches what I see in its reference manual.

<details><summary>LPSPI1_SDO_SELECT_INPUT documentation in the 1020 RM</summary>
<p>

<img width="775" alt="Screenshot 2023-12-21 at 21 26 32" src="https://github.com/imxrt-rs/imxrt-ral/assets/7933926/8d6fa31c-5ae0-4d72-90ea-a0ef651ff48f">

</p>
</details>

The old text represents the **1011**'s LPSPI1_SDO_SELECT_INPUT fields. Those don't belong in the 1021.